core/tests/protractor/editorAndPlayer.js

```diff
@@ -205,6 +205,7 @@ describe('Full exploration editor', function() {
       editor.setTitle('Testing multiple rules');
       editor.setCategory('Answer Groups');
       editor.setObjective('To assess happiness.');
+      editor.previewSummaryTile();
```

Comment:
Hi @seanlip I'm not sure about this. In the editor.js file it advises that all settings actions be run from a 'runFromSettingsTab' wrapper function which prevents me from factoring out separate tests for each action. I presume it'll be fine to just not use the wrapper function and manually navigate to and from the settings tab, but I hesitate because in editor.js it advises running all settings related functions from runFromSettingsTab wrapper.